lib/zipper.js

```diff
@@ -70,11 +73,28 @@ var _addHandlersToZip = function(zip, configs) {
         zip.file('_utils.js', utilsFile);
 };
 
+var _addInternalsToZip = function(zip, deps) {
+    var internals = deps.filter(function(d) {
+        return d.startsWith(global.constants.INTERNALS_PREPENDING_STR);
+    });
+
+    if(internals.length === 0) return;
+
+    var node = zip.folder(global.constants.INTERNALS_FOLDER_NAME);
+    
+    for(var i in internals) {
+        var internal = internals[i].substr(global.constants.INTERNALS_PREPENDING_STR.length);
+        var fileName = path.basename(internal);
+        node.file('fileName.js', fs.readFileSync(internal));
```

Comment:
are you sure this works ? wouldn't this always add a file name 'fileName.js' to the zip node ? probably you wanted to write fileName